elks/tools/mfs/mkfs.c

```diff
@@ -109,8 +109,13 @@ void cmd_mkfs(char *filename, int argc,char **argv) {
   close_fs(fs);
 }
 
-#define BPB_SecPerTrk	24		/* offset of sectors per track (short)*/
-#define BPB_NumHeads	26		/* offset of number of heads (short)*/
+/* ELKS BPB parameters*/
+#define ELKS_BPB_SecPerTrk	505		/* offset of sectors per track (byte)*/
+#define ELKS_BPB_NumHeads	506		/* offset of number of heads (byte)*/
+
+/* MSDOS FAT parameters (unused at present)*/
```

Comment:
What FAT parameters are doing in a tool for working with the MINIX filesystem ?

Reply:
The FAT parameters were added back when were were talking about a combined MINIX and FAT ELKS BPB boot block. They are unused, would you like me to delete them?

Reply:
Yes please.